### DISCUSSION TITLE
Improving documentation

### DISCUSSION THREAD

**Discussion by F43nd1r** [ACRA/acra#869] [General]
After 75 answers to the [User survey](https://goo.gl/forms/nyt9qkCk1GptRGlw2), I've looked at summary. According to you, ACRAs weakest point is its documentation.
Please help improving it by 
- writing down what you found out (anybody can edit the wiki!)
- noting shortcomings of the documentation in a comment here, I'll try to adress them all.

If you're interested in more statistics from the user survey, and conclusions I drew from them, read the next comment.

**Comment by F43nd1r** [comment]
Scoped Questions: 
- About 5% of users are using minsdk versions lower than our current 14. This seems acceptable.
- Increasing the minsdk by a meaningful amount (to 19) would exclude about half of ACRAs users. This is not going to happen anytime soon.
- ACRA 5 has been well received - more people are using it than not. Great :)
- most people are using the http post sender, only a few are using http put. But as these two share almost all of their code, they won't be split up anyways.
- Both http and mailsender are outclassing custom senders by orders of magnitude, which means the customization options seem to be in great shape.
- While the notification interaction is used the least (25%), it is still used enough that all interactions will continue to be actively supported.
- No ReportField that was asked for is used by less than 10%, but most users are just using a few. Maybe splitting off some collectors into separate packages could reduce package size a bit - but probably not worth the effort.
- ACRAs most compelling points are it being open source and free - two thing that definetly won't change.

Freeform Feedback:
- release information can be found on the [Releases Page](https://github.com/ACRA/acra/releases) since 5.2.0 - thanks for noting that this would help.
- some changes to the API are necessary to make new things possible - sorry about that. No big changes like 5.0 are coming though.
- a sample app will probably be added in the future - maybe in a new project? everybody is welcome to add code samples to the wiki.
- Multiple Questions regarding backends - the only thing I can say to that is: I am actively developing and using Acrarium. Other projects like Acralyzer are not my scope. Unless someone else works on them, no updates will come.
- A mail sender which requires no user input sounds great in theory, but is flawed in practice. I didn't find a way to build one without including a password in the app, which would allow anyone who decompiles your app to send e.g. spam from the crash reporting account. Contact me if you know of alternatives.

Do you have more questions?
Ask them here on github where I can respond, not in a feedback box. 

Thanks to everyone who took or will take part in the survey.

[a comment by an automated account is not shown]

**Comment by mage1k99** [comment]
@F43nd1r I'm interested, pleased in helping this.

**Comment by F43nd1r** [comment]
@mage1k99 you mean you're interested in improving documentation? The wiki is open to edits, you can make changes where you see shortcomings.

**Comment by mage1k99** [comment]
And  helping in maintaining (direct contributor ?)

**Comment by F43nd1r** [comment]
@mage1k99 Pull requests are always welcome.
